src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -117,6 +117,7 @@ static void print_help_messages() {
     message_list.emplace_back("OV_GPU_DumpLayersResult", "Dump output buffers of result layers only");
     message_list.emplace_back("OV_GPU_DumpLayersDstOnly", "Dump only output of layers");
     message_list.emplace_back("OV_GPU_DumpLayersLimitBatch", "Limit the size of batch to dump");
+    message_list.emplace_back("OV_GPU_DumpLayersRaw", "Dump memory of layers. If true, dump data is not formated.");
```

Comment:
`When dumping memory, store in raw memory format`

Reply:
Done.